Chaincase/Views/MnemonicPage.xaml.cs

```diff
@@ -16,8 +16,11 @@ public MnemonicPage()
 				this.BindCommand(ViewModel,
 					vm => vm.AcceptCommand,
 					v => v.Accept)
-					.DisposeWith(disposables);
-			});
+					.DisposeWith(disposables);
+
+                this.Bind(ViewModel, x => x.MnemonicString, x => x.MnemonicStringLabel.Text)

```

Comment:
The view doesn't change the MnemonicString here, so this can be `OneWayBind` instead of [TwoWay]`Bind`

Reply:
Makes sense